src/Nethermind/Nethermind.Runner/InitConfig.cs

```diff
@@ -36,5 +36,12 @@ public class InitConfig : IInitConfig
         public bool StoreReceipts { get; set; } = true;
         public DiagnosticMode DiagnosticMode { get; set; } = DiagnosticMode.None;
         public string RpcDbUrl { get; set; } = String.Empty;
+
+        [Obsolete("Use DiagnosticMode with MemDb instead")]
+        public bool UseMemDb
+        {
+            get => DiagnosticMode == DiagnosticMode.MemDb;
+            set => DiagnosticMode = DiagnosticMode.MemDb;
```

Comment:
risk if overwriting when both set

Reply:
Yes but this is backward compatibility of an diagnostic switch. This can be ignored by design. I assume that person who does diagnostics more or less knows what is doing.